### DISCUSSION TITLE
Termination issues with new PCB

### DISCUSSION THREAD

**Discussion by pmcmick** [BlueSCSI/BlueSCSI-v2#137] [Help]
Hi, 

I've just finished a build on a new PCB from JLCPCB.  In a nutshell, even though I have the jumpers all set appropriately the BlueSCSI isn't showing up.  I have the power jumper set, termination set, and target mode set.  The board rev is desktop V2 2023.10a.  I just updated the firmware and it is version: 2024.04.02-release Apr  3 2024 02:22:50.  The JLCPCB SMD part went OK in terms of components -- everything was in stock, so no replacements.  

Everything in the log shows the image is ready (there is only one on the SD card) but I see zero activity on the SCSI activity LED.  Extra debug output shows: 

> [232ms] DBG -- SelectionDelay: 255
[233ms] DBG -- EnableUnitAttention is off
[233ms] DBG -- EnableSCSI2 is on
[233ms] DBG -- EnableSelLatch is off
[233ms] DBG -- MapLunsToIDs is off
[233ms] -- Debug is enabled
[233ms] DBG -- Parity is enabled
[233ms] DBG -- ReinsertCDOnInquiry is disabled
...
[319ms] DBG ---- Read prefetch enabled: 8192 bytes
[321ms] DBG ---- Valid Macintosh Device Image detected.
...
[1458ms] DBG BUS RESET

When I boot up from a floppy and run SCSIProbe, it reports that the bus is not terminated (device ID 7 is the only one to show -- the BlueSCSI is the only device connected to the system).  I've checked continuity on the jumper lines and any other minor checks I could come up with.   I didn't see anything obviously wrong but didn't go all the way to digging into individual chips and their placements (beyond the Pico).  

I've put a different (earlier rev) BlueSCSI board into the same system, same SD card, same cables, and the same power connections.  It works perfectly.  

Any suggestions on where to take my debugging next?  I'm certainly leaning towards something wrong on the board itself but I'm hoping I've overlooked something obvious.  

Thanks for any suggestions and help you may have.

**Comment by erichelgeson** [comment]
How does your SCSI chain look? Do you have a device/terminator you could use as termination instead of BlueSCSI (just to rule in/out any issue there?) I'd still run through the Troubleshooting guide in the wiki just to rule out easy software things before spending time on the hardware items (just always good to do to rule it out)

I know you said the parts were all in stock but I havent seen that at jlcpcb for a while, always something in or out - one thing recently was the 1% vs 5% termination resistors - people were wondering if they could swap - you can not.

cc @androda

**Reply by pmcmick** [reply]
Thanks @erichelgeson -- yep, I went over the entire troubleshooting guide yet again last night.  No joy...   Given I have an older board rev that works in the same exact setup, I decided to build a 2nd board from the JLCPCB batch to rule out a board-specific issue. It just didn't feel like software or something simple to me -- just a hunch...  The 2nd board works just fine.

Not sure how much effort to put into tracking down the problem(s) with the 1st board but the tinkerer in me wants to chase it down.   I've never had a bad board from JLCPCB so it would be good to know if I indeed got one this go 'round.  

Thanks again.
